scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/RMProxy.java

```diff
@@ -60,6 +61,8 @@
 
     private Credentials creds;
 
+    private Map<String, HybridEncryptionUtil.HybridEncryptedData> thirdPartyCredentials;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "thirdPartyCredentials" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)

Reply:
same as sonar said, remove it if not used